components/identities/identityCard.tsx

```diff
@@ -123,13 +121,6 @@ const IdentityCard: FunctionComponent<IdentityCardProps> = ({
               <div className="flex flex-col">
                 {identity?.targetAddress ? (
                   <>
-                    <div className={styles.addressBar}>
-                      <h2>{minifyAddress(identity.targetAddress)}</h2>
-                      <CopyContent
-                        value={identity?.targetAddress}
-                        className="cursor-pointer ml-3"
-                      />
-                    </div>
```

Comment:
Nope, here you delete the address for every identity. We want to delete the address for the virgin identities only. For the domains we want to keep them :)